docs/source/telegram.transactionpartnertelegramads.rst

```diff
@@ -4,4 +4,4 @@ TransactionPartnerTelegramAds
 .. autoclass:: telegram.TransactionPartnerTelegramAds
     :members:
     :show-inheritance:
-    :inherited-members: TransactionPartner
```

Comment:
the docs of `TransactionPartnerTelegramAds` now also show the class variables of `TransactionPartner`. That's not desired IMO.